code/testapp/src/main/res/layout/activity_main.xml

```diff
@@ -51,10 +51,10 @@
         android:layout_marginStart="56dp"
         android:layout_marginLeft="60dp"
         android:layout_marginTop="24dp"
-        android:onClick="trackQueue"
+        android:onClick="getQueueSize"
         android:text="Get Queue Size"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/setIdentifier" />
+        app:layout_constraintTop_toBottomOf="@+id/setVid" />
```

Comment:
optout button id needs to be updated

current: L105 `app:layout_constraintTop_toBottomOf="@+id/unknown"`

Reply:
The button ID looks ok to me android:id="@+id/optOut", and the line you shared is the layout top constraint, which is also correct (under button unknown). Am I missing anything?

Reply:
<img width="1101" alt="Screenshot 2023-02-17 at 11 53 57 AM" src="https://user-images.githubusercontent.com/4697559/219778088-acfe7d3b-5186-4e18-9dd2-63668e267536.png">